examples/gcoap/client.c

```diff
@@ -418,7 +390,8 @@ int gcoap_cli_cmd(int argc, char **argv)
 
         printf("gcoap_cli: sending msg ID %u, %" PRIuSIZE " bytes\n",
                 coap_get_id(&pdu), len);
-        if (!_send(&buf[0], len, _proxied ? &_proxy_remote : &remote)) {
+        if (_send(&buf[0], len, *_proxy_uri ? &_proxy_remote : &remote, NULL,
+                   *_proxy_uri ? _get_tl(_proxy_uri) : _get_tl(_last_req_uri)) <= 0) {
```

Comment:
I think the readability suffers too much. Could you split that into a regular `if`?